vendingMachine/src/snackBarApp/Snack.java

```diff
@@ -0,0 +1,79 @@
+package snackBarApp;;
+
+public class Snack
+{
+    private static int foodId = 0;
```

Comment:
since were inside a snack object we dont really need to explicitly say that its a foodId, we can just say id becuase its an id property inside of a snack, meaning its that snacks id.
```suggestion
    private static int id = 0;
    //or
    private static int maxId = 0;
```